x-pack/metricbeat/module/aws/dynamodb/_meta/fields.yml

```diff
@@ -4,3 +4,137 @@
     `dynamodb` contains the metrics that were scraped from AWS CloudWatch which contains monitoring metrics sent by AWS DynamoDB.
   release: beta
   fields:
+    - name: metrics
+      type: group
+      fields:
+        - name: SuccessfulRequestLatency
+          type: group
+          description: >
+            The latency average of successful requests to DynamoDB or Amazon DynamoDB
+            Streams during the specified time period.
+          fields:
+            - name: avg
+              type: long
```

Comment:
Should some of these metrics be `double` type? For example, in data.json `aws.dynamodb.metrics.SuccessfulRequestLatency.avg` = 1.6592021822660081

Reply:
+1, I think that most of the metrics here should be double.

Reply:
😬 nice catch, thank you both!